### ISSUE TITLE
Virgin Media (ISP) is blocking nfts stored on ipfs.nftstorage.link and  https://nftstorage.link

### ISSUE THREAD

**Issue by PennilessImp749** [nftstorage/nft.storage#1946]
Okay so I have been struggling for some time, where I have invested in about 6-8 projects, and have their NFTs in my wallet.
2 of those collections were not showing up, and I thought it was my router or connection,
![issue](https://user-images.githubusercontent.com/35653102/170474619-87d28994-a675-42f5-b08c-137e9420074d.png)

However after a few weeks i dug a little deeper, and it turns out that Virgin Media is blocking ipfs.nftstorage.link and  https://nftstorage.link/ . 
![Screenshot 2022-05-26 120019](https://user-images.githubusercontent.com/35653102/170475311-32e4d773-7902-40ef-ba73-99d331033de6.png)
The only way that I could fix this issue was to log into my ISP account (where i pay my bills etc) and go into Manage your Web Settings, and turn off Virus Safe which was on by default.
![fix](https://user-images.githubusercontent.com/35653102/170474715-8cfb9031-83e1-49bf-a4c5-021cac0a37b8.png)

Virus Safe is a feature for Virgin media : https://www.virginmedia.com/broadband/parental-control
Virus Safe prevents you from accessing web sites which may be fraudulent, or contain harmful malware and viruses.

I am guessing that Virgin Media has flagged your website that you host the NFTs for the same thing that malwarebites has blocked for "Malwarebytes blocks the subdomain ipfs.nftstorage.link of the domain nftstorage.link because it is associated with [phishing](https://blog.malwarebytes.com/glossary/phishing/)."

Perhaps Virgin Media has flagged it for the same reason or something but if I have this issue then everyone who is in the United Kingdom, who uses virgin media which 3rd biggest ISP in the United Kingdom with 5,595,800 Subscribers. Anyone with NFTs will be affected by this.

Yes I was able to fix this by turning off the Virus Safe option in my account settings, because I figured out what the issue was but if you could somehow get them to whitelist your services, a lot of people will have the issue fixed without them having to do anything. I really had to dig deep to figure out where the address of my NFT was on the blockchain to get that address and look at it using my ISP connection, had I not done this I would not have figured this out.

I wish I could assist you further with this or explain further, I hope I was able to express the issue, and hopefully there can be a solution soon, as this will be affecting a multitude of your users.

Thanks

Andrew (aka PennilessImp749)

**Comment by dchoi27** [comment]
Yes, we reached out yesterday! If you're able to support this getting resolved as quickly as possible by filling out the "Don't think this page should be blocked? Let us know." form that would be great.

**Comment by PennilessImp749** [comment]
![Screenshot 2022-05-27 211332](https://user-images.githubusercontent.com/35653102/170783332-b2569791-ff4f-465b-b37b-02b891c7f26a.png)
Okay I was able to report that http://ipfs.nftstorage.link and https://nftstorage.link should not be blocked.
Hopefully this will solve the problem with my ISP. However, I believe there are many other countries and ISPs, that are suffering the same, from what I gather from various discord channels. I wish you all well and I was glad to bring this to your attention.

**Comment by PennilessImp749** [comment]
Do I have to close this thread or do you? Sorry I am new to github pretty much.

**Comment by dchoi27** [comment]
Thanks for closing! I think the issue's been resolved now, but would appreciate if you have a spare second to confirm.

Agreed that it's a bit like playing whack-a-mole - part of the bumps in the road going from web2 to web3. We had a blogpost on this topic: https://nft.storage/blog/post/2022-04-29-gateways-and-gatekeepers/

If you see any others with this issue in the wild, please let them know to open an issue here! Appreciate it

**Comment by PennilessImp749** [comment]
Thats a fascinating read thank you @dchoi27 for sharing the link and the transparency. Yes I can confirm that the issue has been resolved for me personally, I turned on the Web Safe feature on my ISP, and the NFTs and the Direct Link to nftstorage.link is allowed and things are showing and appearing as normal. I am sorry that you are going through this nightmare, and hope things go more smoothly from here on out. Thank you so much for your assistance.
